Assets/loadScene.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class loadScene : MonoBehaviour
```

Comment:
Probably should change this script name from loadScene to something like UIButtonMethods, since the script does more than just load a scene now.